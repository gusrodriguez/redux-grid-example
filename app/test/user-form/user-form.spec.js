/* eslint-disable */ //disable lint because 'describe' does not need to be defined in this scope.
const React = require('react');
const { shallow, mount, render } = require('enzyme');
const USerFormRedux = require('../../shared/components/user-form');
const renderer = require('react-test-renderer');

describe('UserForm component tests', () => {
    const UserForm = USerFormRedux.__get__('UserForm');

  it('should render correctly', () => {
    const output = shallow(<UserForm />);
    expect(output).toMatchSnapshot();
  });
});