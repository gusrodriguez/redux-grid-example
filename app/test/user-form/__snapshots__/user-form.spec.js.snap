// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserForm component tests should render correctly 1`] = `
<form
  className="container user-form"
  onSubmit={[Function]}
>
  <div
    className="row form-row"
  >
    <div
      className="col-md-12"
    />
  </div>
  <div
    className="row form-row"
  >
    <div
      className="col-md-6"
    >
      <input
        className="form-control"
        maxLength="50"
        maxlength="50"
        name="firstname"
        pattern="[a-zA-Z0-9]+"
        placeholder="First name"
        size="50"
        type="text"
      />
    </div>
    <div
      className="col-md-6"
    >
      <input
        className="form-control"
        maxLength="50"
        maxlength="50"
        name="surname"
        pattern="[a-zA-Z0-9]+"
        placeholder="Surname"
        size="50"
        type="text"
      />
    </div>
  </div>
  <div
    className="row form-row"
  >
    <div
      className="col-md-6"
    >
      <input
        className="form-control"
        maxLength="50"
        name="contact_number"
        pattern="^[0-9+]*$"
        placeholder="Contact Number"
        size="50"
        type="text"
      />
    </div>
    <div
      className="col-md-6"
    >
      <input
        className="form-control"
        maxLength="50"
        name="email"
        pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\\\\.[a-z]{2,4}$"
        placeholder="Email"
        size="50"
        type="email"
      />
    </div>
  </div>
  <div
    className="row form-row"
  >
    <div
      className="col-md-12"
    >
      <button
        className="btn btn-primary pull-right"
        type="submit"
      >
        Add
      </button>
    </div>
  </div>
</form>
`;
